src/sentry/search/utils.py

```diff
@@ -39,14 +40,15 @@ def get_user_tag(project, key, value):
         )[0]
     except (KeyError, IndexError):
         return u'{}:{}'.format(key, value)
-
+    except DataError:
+        raise InvalidQuery(u'malformed \'{}:\' query \'{}\'.'.format(key, value))
```

Comment:
It's generally stylistically suggested to avoid escaping characters if you can avoid it. So I'd use double quotes so the single quotes don't need escaped.